EndlessClient/Subscribers/GuildEventSubscriber.cs

```diff
@@ -0,0 +1,54 @@
+﻿using AutomaticTypeMapper;
+using EndlessClient.Audio;
+using EndlessClient.Dialogs.Factories;
+using EOLib.Domain.Interact.Guild;
+using EOLib.Domain.Notifiers;
+using EOLib.Localization;
+using EOLib.Net.Communication;
+using Moffat.EndlessOnline.SDK.Protocol.Net.Client;
+
+namespace EndlessClient.Subscribers
+{
+    [AutoMappedType]
+    public class GuildEventSubscriber : IGuildNotifier
+    {
+        private readonly IEOMessageBoxFactory _messageBoxFactory;
+        private readonly ILocalizedStringFinder _localizedStringFinder;
+        private readonly IPacketSendService _packetSendService;
+        private readonly ISfxPlayer _sfxPlayer;
+
+        public GuildEventSubscriber(IEOMessageBoxFactory messageBoxFactory, ILocalizedStringFinder localizedStringFinder, IPacketSendService packetSendService, ISfxPlayer sfxPlayer)
+        {
+            _messageBoxFactory = messageBoxFactory;
+            _localizedStringFinder = localizedStringFinder;
+            _packetSendService = packetSendService;
+            _sfxPlayer = sfxPlayer;
+        }
+
+        public void NotifyGuildCreationRequest(GuildCreationRequest request)
+        {
+            _sfxPlayer.PlaySfx(SoundEffectID.ServerMessage);
+
+            var dlg = _messageBoxFactory.CreateMessageBox(
+                $"{request.GuildIdentity}" + " " +
```

Comment:
Minor nit: formatting. Put spaces as the first part of the line, indent subsequent lines for the string, comma does not start the `caption` line

```cs
$"{request.GuildIdentity}"
    + " " + _localizedStringFinder.GetString(...)
    + " " + ...
    + " " + _localizedStringFinder.GetString(EOResourceID.GUILD_DO_YOU_ACCEPT),
caption: ...
```